lib/fs.js

```diff
@@ -207,6 +207,15 @@ fs.accessSync = function(path, mode) {
   binding.access(pathModule._makeLong(path), mode);
 };
 
+fs.accessibleSync = function(path, mode) {
+  try {
+    fs.accessSync(path, mode);
```

Comment:
For the sake of being explicit I'd make sure that if `mode === undefined` then you pass `fs.F_OK`.